packages/api/src/external/commonwell/document/document-query-sandbox.ts

```diff
@@ -95,6 +104,7 @@ export async function sandboxGetDocRefsAndUpsert({
         }
       : undefined),
     requestId,
+    source: MedicalDataSource.COMMONWELL,
```

Comment:
ditto: having `source` optional and just update the patient's global status instead?